common/src/main/java/tachyon/exception/TachyonException.java

```diff
@@ -57,19 +54,47 @@ public TachyonTException toTachyonTException() {
     return new TachyonTException(mType.name(), getMessage());
   }
 
+  /**
+   * Construct a TachyonException from a TachyonTException.
+   *
+   * @param e The TachyonTException to convert to a TachyonException
```

Comment:
our style is either `The TachyonTException to convert to a TachyonException.` (preferred) or `the TachyonTException to convert to a TachyonException` (either start with a uppercase and end with a period, or start with lowercase and end without a period).

Also, for class name references use `{&amp;link Class}` (e.g. `{&amp;link TachyonException}`